packages/components/modal/stories/ModalLauncher.stories.tsx

```diff
@@ -1,12 +1,11 @@
 import React, { useState } from 'react';
-import { storiesOf } from '@storybook/react';
 import { Paragraph } from '@contentful/f36-typography';
 
-import { ModalLauncher } from './ModalLauncher';
+import { ModalLauncher } from '../src/ModalLauncher/ModalLauncher';
 import { Button } from '@contentful/f36-button';
-import { Modal } from '../Modal';
+import { Modal } from '../src/Modal';
 
-function DefaultStory() {
+export const Basic = () => {
```

Comment:
This story is not showing up, and I think it is because it's missing the default export with storybook setup you know?

The one that looks like this:

```js
export default {
  title: 'Utilities/ModalLauncher',
  component: ModalLauncher,
  // ...
} as Meta;
```

Reply:
thanks for the suggestion, done